scapy/sendrecv.py

```diff
@@ -79,14 +90,12 @@ class _BreakException(Exception):
     pass
 
 
-def _sndrcv_rcv(pks, tobesent, stopevent, nbrecv, notans, verbose, chainCC,
-                multi):
+def _sndrcv_rcv(pks, hsent, stopevent, nbrecv, notans, verbose, chainCC,
+                multi, _storage_policy=None):
     """Function used to recieve packets and check their hashret"""
+    if not _storage_policy:
```

Comment:
`if _storage_policy is None:` will be nicer.

Also could you add a comment explaining the purpose of this function.